lib/CompUnit/Repository/Staging.rakumod

```diff
@@ -4,6 +4,7 @@ class CompUnit::Repository::Staging is CompUnit::Repository::Installation {
 
     submethod TWEAK(--> Nil) {
         $!parent = CompUnit::RepositoryRegistry.repository-for-name($!name);
+        self.copy-config-from($!parent);
```

Comment:
If I understand this correctly, the Staging repo directory will then contain a config file. This will create conflicts when multiple module packages then contain a config file for the same repository. The end result of an installation into a Staging repository should be a directory that can then be packaged as-is. Think: "I can tar this directory and un-tar it on a different machine and the module will then be installed there"

Reply:
Good point. I didn't realize it would try to copy the config over as well. That should not be.

On a related note, are we guaranteed, that the repo we copy the staged modules into will have the same configuration as the staging repos parent? Or phrased differently, can we accept the limitation that the final module destination repo will have to be identically configured to the staging repos parent? (In the typical case of a linux distro packaging up modules as well as the zef install case, it should work. But if you want to deploy a tared module to an unrelated rakudo installation on a different machine, it will likely break.)

Reply:
I've now made it so the staging repo has no config itself, but relays to the parent repos config. That's a pretty clean solution in my opinion.